x-pack/plugins/secops/public/components/notes/note_cards/index.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiPanel } from '@elastic/eui';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { Note } from '../../../lib/note';
+import { AddNote } from '../add_note';
+import { AssociateNote, GetNewNoteId, UpdateNote } from '../helpers';
+import { NoteCard } from '../note_card';
+
+const AddNoteContainer = styled.div`
+  padding: 5px;
+`;
+
+const NoteContainer = styled.div`
+  margin-top: 5px;
+`;
+
+const NoteCardsContainer = styled(EuiPanel)<{ width?: string }>`
```

Comment:
Can you use a default if this is undefined as in `width?: string = 100%`? Then you wouldn't need the `?` below